core/src/trezor/messages/CardanoTxMetadataType.py

```diff
@@ -0,0 +1,34 @@
+# Automatically generated by pb2py
+# fmt: off
+import protobuf as p
+
+from .CardanoCatalystRegistrationParametersType import (
```

Comment:
the formatting does not match what `make gen` produces

Reply:
Yes, is there something I can do about this? Running `make pystyle` reformats the file.

Although maybe it's irrelevant since the file is going to be removed.

Reply:
probably :) `make pystyle` should not be touching the file since it's explicitly saying `# fmt: off`
so that might be a bug in your local `black` copy or something?

Reply:
oh, but it's `isort` doing this. weird.

Reply:
The problem is still there but moved to `core/src/trezor/messages/CardanoTxAuxiliaryDataType.py`. Is there something I should do about it?

Reply:
I think if you stuff `# isort:skip_file` into the pb2py template this will go away.

Reply:
Done in https://github.com/trezor/trezor-firmware/pull/1564/commits/5eb535fbe212ee55bed20d11ee0bc450065a09eb. Interestingly enough `# isort: skip_file` (with a space after `:`) didn't seem to work although that's what's used in the [isort docs](https://pycqa.github.io/isort/docs/configuration/action_comments/) - maybe different version?